server/odc-service/src/main/java/com/oceanbase/odc/service/collaboration/project/ProjectService.java

```diff
@@ -193,6 +192,34 @@ public void createProjectIfNotExists(@NotNull User user) {
         userResourceRoleRepository.batchCreate(userResourceRoleEntities);
     }
 
+    /**
+     * Create a built-in project for migrate user
+     *
+     * @param user user
+     */
+    @SkipAuthorize("odc migrate usage")
+    @Transactional(rollbackFor = Exception.class)
+    public ProjectEntity createProject4Migrate(@NotNull User user) {
+        // default name
+        String projectName = BUILTIN_PROJECT_PREFIX + "INNER";
+        // if project exist
+        if (repository.findByNameAndOrganizationId(projectName, user.getOrganizationId()).isPresent()) {
```

Comment:
if exists, return the existing one rather than null

Reply:
Got it

Reply:
if exists, there is no need to migrate. Is it better to return to null?

Reply:
if exists, there is no need to migrate. Is it better to return to null?